gateway/src/node/statistics/mod.rs

```diff
@@ -1,4 +1,63 @@
 // Copyright 2022 - Nym Technologies SA <contact@nymtech.net>
 // SPDX-License-Identifier: GPL-3.0-only
 
-pub mod collector;
+use futures::{channel::mpsc, StreamExt};
+use nym_node_http_api::state::metrics::SharedSessionStats;
+use nym_statistics_common::events::{StatsEvent, StatsEventReceiver, StatsEventSender};
+use nym_task::TaskClient;
+use sessions::SessionStatsHandler;
+use std::time::Duration;
+use time::OffsetDateTime;
+use tracing::trace;
+
+pub mod sessions;
+
+const STATISTICS_UPDATE_TIMER_INTERVAL: Duration = Duration::from_secs(3600); //update timer, no need to check everytime
+
+pub(crate) struct GatewayStatisticsCollector {
+    stats_event_rx: StatsEventReceiver,
+    session_stats: SessionStatsHandler,
+    //here go additionnal stats handler
```

Comment:
nit: spelling

Reply:
Fixed in adf198f6b8696b963643253dd0ee3aa6e745f13c